azurerm/internal/services/network/tests/web_application_firewall_policy_data_source_test.go

```diff
@@ -0,0 +1,127 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+)
+
+func TestAccDataSourceAzureRMWebApplicationFirewallPolicy_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "data.azurerm_web_application_firewall_policy", "test")
+	resourceGroupName := fmt.Sprintf("acctestRG-%d", data.RandomInteger)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:  func() { acceptance.PreCheck(t) },
+		Providers: acceptance.SupportedProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceAzureRMWebApplicationFirewallPolicyBasic(data),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(data.ResourceName, "name", "example-wafpolicy"),
+					resource.TestCheckResourceAttr(data.ResourceName, "resource_group_name", resourceGroupName),
+					resource.TestCheckResourceAttr(data.ResourceName, "tags.%", "1"),
```

Comment:
Test errors here as the resource doesn't have any tags set. If we're reusing the configuration from the resource test, we could add some there.

azurerm/internal/services/network/tests/web_application_firewall_policy_resource_test.go  L271
```hcl

  tags = {
    env = "test"
  }

```